frontend/src/views/applications/ApplicationsHistoryView.vue

```diff
@@ -367,28 +371,26 @@ export default {
       })
       return applicationItems
     },
-
-    //TODO - Add Supp App PDF function will look very similar, but it will hit a seperate endpoint
-    //the supp app PDF's do not generate yet in Dynamics, so holding off on including that code
     async downloadPDF(application) {
-      if (application.applicationType === APPLICATION_TYPES.OFM) {
-        try {
-          const resp = await ApplicationService.getApplicationPDF(application.applicationId)
-          const link = document.createElement('a')
-
-          link.href = `data:application/pdf;base64,${resp}`
-          link.target = '_blank'
-          link.download = application.referenceNumber
-          // Simulate a click on the element <a>
-          document.body.appendChild(link)
-          link.click()
-          document.body.removeChild(link)
-        } catch (error) {
-          this.setFailureAlert('Failed to download OFM PDF', error)
+      try {
+        let resp
+        if (application.applicationType === APPLICATION_TYPES.OFM) {
+          resp = await ApplicationService.getApplicationPDF(application.applicationId)
+        } else {
+          resp = await ApplicationService.getSupplementaryApplicationPDF(application.supplementaryApplicationId)
         }
-      } else {
-        //it's a supp app
-        this.setSuccessAlert(`COMING SOON! A supplementary PDF will be downloaded here.`)
+
+        const link = document.createElement('a')
+        link.href = `data:application/pdf;base64,${resp}`
+        link.target = '_blank'
+        link.download = application.referenceNumber
+
+        // Simulate a click on the element <a>
+        document.body.appendChild(link)
+        link.click()
+        document.body.removeChild(link)
+      } catch (error) {
+        this.setWarningAlert('Task is still in progress. Please wait a few minutes before you try again.')
```

Comment:
As discussed "PDF Generation is still..." might be clearer.

Reply:
done!